models/gif_pop_psc_exp.h

```diff
@@ -52,15 +52,15 @@ Description
 
 This model simulates a population of spike-response model neurons with
 multi-timescale adaptation and exponential postsynaptic currents, as
-described in [1]_.
+described in Schwalger et al. (2017) [1]_.
 
-The single neuron model is defined by the hazard function
+The single neuron model is defined by the hazard function:
 
 .. math::
 
  lambda_0 * exp[ ( V_m - E_sfa ) / Delta_V ]
```

Comment:
```suggestion
 \lambda_0 * \exp\left(( V_m - E_{sfa} ) / \Delta_V\right)
```

Reply:
Good point, @clinssen, we could maybe think about a general rule which we can apply to all models. This one I would leave as it is now and fix it if necessary after the discussion.

Reply:
We cannot leave it as-is, as it wouldn't render right; would have to escape the underscores in that case and wrap variable names in \texttt{}.